superdesk/sams/decorator.py

```diff
@@ -0,0 +1,14 @@
+from functools import wraps
+from flask import request
+
+
+def blueprint_auth(app):
```

Comment:
Can you add some comments here as to what this decorator is used for.
Also, this file should probably reside under `superdesk/auth/decorator.py` so all blueprints have access to this generic auth functionality

Reply:
Comments added and file moved to superdesk/auth/decorator.py